File: src/backend/src/main/java/com/wissen/SmartInterviewProcess/models/Employee.java
package com.wissen.SmartInterviewProcess.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Employee {

	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private long id;
	private String name;
	private String phoneNumber;
	private String email;
	private String wissenId;
	
	private boolean active=false;

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getPhoneNumber() {
		return phoneNumber;
	}

	public void setPhoneNumber(String phoneNumber) {
		this.phoneNumber = phoneNumber;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getWissenId() {
		return wissenId;
	}

	public void setWissenId(String wissenId) {
		this.wissenId = wissenId;
	}

	public boolean isActive() {
		return active;
	}

	public void setActive(boolean active) {
		this.active = active;
	}

	@Override
	public String toString() {
		return "Employee [id=" + id + ", name=" + name + ", phoneNumber=" + phoneNumber + ", email=" + email
				+ ", wissenId=" + wissenId + ", active=" + active + "]";
	}
	
	
	
}

File: src/app/models/Employee.ts
export default class Employee {

    private email: string;
    private name: string;
    private wissenId: string;
    private phoneNumber: string;
    private role : string;



    constructor(email,name,wissenId,phoneNumber,role){
        this.email = email;
        this.name = name;
        this.wissenId = wissenId;
        this.phoneNumber = phoneNumber;
        this.role = role;
    }

    get getRole() : string{
        return this.role;
    }
    get getEmail(): string {
        return this.email;
    }

    get getName(): string {
        return this.name;
    }

    get getwissenId(): string {
        return this.wissenId;
    }

    get getPhoneNum(): string {
        return this.phoneNumber;
    }

    set setEmail(email: string) {
        this.email = email;
    }

    set setName(name: string) {
        this.name = name;
    }

    set setWissenId(id: string) {
        this.wissenId = id;
    }

    set setPhoneNum(num: string) {
        this.phoneNumber = num;
    }
    set setRole(role:string) {
        this.role =role;
    }
}